### ISSUE TITLE
"G153 Making the text easier to read" needs referral to "lower secondary education level"?

### ISSUE THREAD

**Issue by jake-abma** [w3c/wcag#1753]
G153 is about Making the text easier to read and refers to: Success Criterion 3.1.5: Reading Level, even as "(Sufficient)"

But the fixes are not the same as the condition for Success Criterion 3.1.5: Reading Level, namely the "lower secondary education level" part.

So, OR add "lower secondary education level" in the technique OR do not make it a Sufficient technique, and probably change name as it is confusing as it kind of suggest to fix the same problem.

https://www.w3.org/WAI/WCAG22/Understanding/reading-level
https://www.w3.org/WAI/WCAG22/Techniques/general/G153

**Comment by alastc** [comment]
The test procedure for G153 says "Check that the text requires reading ability less advanced than the lower secondary education level."

To me the weakness of the technique is that it just says "Measure the readability of the text.", without saying how.

However, making the text easier to read should achieve the "lower secondary education level", so I don't think this is an issue.

**Comment by jake-abma** [comment]
but in the "In order to reduce the complexity of the text:" it doesn't say that for the "lower secondary education level".
Only the...

> "Use sentences that are no longer than the typical accepted length for secondary education. (Note: In English that is 25 words.)"

... mentioned it

And the procedure says:

> Measure the readability of the text.

And indeed... how?

**Comment by alastc** [comment]
The listing of things which bring down the reading level is how you achieve "lower secondary education level". As there is no one thing that will guarantee you meet the threshold, it cannot say which will do so.

I think you've opened another issue about the measurement, I don't think this is an issue, or at least not one that is worth spending time on.

**Comment by jake-abma** [comment]
ok, will close, thx
